lib/bundler/cli.rb

```diff
@@ -206,6 +206,12 @@ def install
       "Update ruby specified in Gemfile.lock"
     method_option "bundler", :type => :string, :lazy_default => "> 0.a", :banner =>
       "Update the locked version of bundler"
+    method_option "patch", :type => :boolean, :banner =>
+      "Prefer updating only to next patch version"
```

Comment:
there's a `--strict` flag I hadn't added here yet I just realized ... 

If `--strict` is also used, then the code will actually remove versions fed back to Molinillo, but that can result in no updates happening or VersionConflict cases that report a version can't be found that actually does exist in the index, causing a very weird "looks like a bug" experience. (hmmm ... while I did see VersionConflict cases early on, I haven't seen one in a while ... I may need to try and recreate that in a spec -- maybe that was due to an early bug and shouldn't be possible). 

So, not using `--strict` is the default and it means the code will only sort to the front (not filter out) the patch versions first - which favors getting something done, but it's potentially not _as_ conservative. 

One challenge still existing is trying to give better output in these conservative cases, because sometimes no change will happen at all and there's no feedback to the user trying to explain "well, you wanted to go up to 1.8.3 from 1.8.2, but 1.8.3 needs a minor increment and you fed it --strict so it couldn't do anything" - but I've no idea if that's even plausible. It wouldn't be too unlike the Conflict output ... but these aren't cases of a Conflict.

I did think also there could be a way to detect increments beyond what was requested and warn those somehow. 

#pascalsapology